zboss/README.rst

```diff
@@ -7,6 +7,13 @@ The |NCS|'s Zigbee stack uses ZBOSS – a portable, high-performance Zigbee soft
 
 For detailed documentation of the ZBOSS API and instructions on how to use it, see `ZBOSS API documentation`_.
 
+There are two versions of ZBOSS libraries available in the nrfxlib repository:
+
+* Production-ready version, placed under the production subdirectory - containing the latest stable, certification-ready libraries.
```

Comment:
```Production-ready``` --> ```production```.

... ```containing the latest stable ZBOSS libraries. Production libraries are always tested in-house by Nordic Semiconductor to conform to Zigbee Pro R22 test specification. However, production libraries are only periodically verified by external test service provider and granted Zigbee Compliant Platform certification by Zigbee Allicance. Check <<link to compatiblity matrix>> for certification status of production libraries distributed in a particular NCS release.```

Reply:
What compatibility matrix is meant here?

Reply:
Note added in the new section `ZBOSS library certification` inside `zboss/doc/zboss_configuration.rst`.